required/amsmath/testfiles/amsldoc-4-fleqn.lvt

```diff
@@ -0,0 +1,53 @@
+\documentclass[fleqn]{article}
+
+
+\usepackage[alignedleftspaceyes]{amsmath}
+\input{test2e}
+\showoutput
+\begin{document}
+
+\START
+
+aligned
+\begin{equation*}
+\left.\begin{aligned}
+  B'&=-\partial\times E,\\
+  E'&=\partial\times B - 4\pi j,
+\end{aligned}
+\right\}
+\qquad \text{Maxwell's equations}
+\end{equation*}
+
+cases
+\begin{equation}\label{eq:C}
+P_{r-j}=
+  \begin{cases}
+    0&  \text{if $r-j$ is odd},\\
+    r!\,(-1)^{(r-j)/2}&  \text{if $r-j$ is even}.
+  \end{cases}
+\end{equation}
+
+intertext
+\begin{align}
+A_1&=N_0(\lambda;\Omega')-\phi(\lambda;\Omega'),\\
+A_2&=\phi(\lambda;\Omega')-\phi(\lambda;\Omega),\\
+\intertext{and}
+A_3&=\mathcal{N}(\lambda;\omega).
+\end{align}
+
+smallmatrix
+\begin{math}
+\bigl( \begin{smallmatrix}
+  a&b\\ c&d
+\end{smallmatrix} \bigr)
+\end{math}
+
+pmatrix
+\begin{equation}\label{eq:D}
+\begin{pmatrix} D_1t&-a_{12}t_2&\dots&-a_{1n}t_n\\
+-a_{21}t_1&D_2t&\dots&-a_{2n}t_n\\
+\hdotsfor[2]{4}\\
+-a_{n1}t_1&-a_{n2}t_2&\dots&D_nt\end{pmatrix},
+\end{equation}
+
+\end{document} 
```

Comment:
missing newline at end of file, doesn't hurt but shows up on other platforms.

Reply:
ok then I will delete the other PR and write a paragraph for this one.  But I'm not sure I know what to do about the newline. What does show up here?

Reply:
you don't need to drop the pull requests you can do all the updates in the same one this way we don't generate unnecessary new issues. in the end we can squash all updates into a single merge. As for the missing newline I can fix that if you don't see it. The last line of the file has no newline at the end and editors like emacs flag that as in some programs that can be harmful (TeX doesn't mind)

Reply:
<img width="1440" alt="Screenshot 2021-08-25 at 00 24 44" src="https://user-images.githubusercontent.com/1062981/130698104-7184ef55-a41d-47f4-ac85-e37e13f2a268.png">

SourceTree also warns :-)

Reply:
I'll fixed it